### PR TITLE
Update cities.md

### PR DESCRIPTION
**Pull request by Christewart**
Add Taiwan

### CODE REVIEW

**Comment by btsea** [comment]
Super cool @Christewart! Thank you for sharing. Do you know if they plan to maintain topic lists? It's not a requirement for inclusion, but if they do I would prefer to link to those.

File: cities.md [hunk 1]
```diff
@@ -21,5 +21,6 @@ As a disclaimer, BitDevs NYC has no official associations with or oversight of t
 - [Los Angeles](https://bitdevsla.org/)
 - [Miami](https://miamibitdevs.org/)
 - [San Francisco](https://www.sfbitcoindevs.org/)
+- [Taiwan](https://bitdevs.tw/)
```
Comment (Xekyo) [inline cities.md:24]: If we're going for cities, I think this would be:

```suggestion
- [Taipei](https://bitdevs.tw/)
```

**Comment by Christewart** [comment]
> Super cool @Christewart! Thank you for sharing. Do you know if they plan to maintain topic lists? It's not a requirement for inclusion, but if they do I would prefer to link to those.

I do not know if they will be building a topic list

**Comment by Xekyo** [comment]
Thanks @Christewart

**Pull request merged**